maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -128,6 +143,7 @@ private void export(File runFile, FileSetRepository fileSetRepository) throws Ex
 		try (Bndrun run = Bndrun.createBndrun(null, runFile)) {
 			run.setBase(temporaryDir);
 			Workspace workspace = run.getWorkspace();
+			workspace.setParent(processor);
```

Comment:
I am not sure this is correct. When the Workspace is created, its parent is `Workspace.getDefaults()`. You now replace the parent and lose the defaults.

Reply:
But perhaps the workspace defaults are pointless for a maven project.

https://github.com/bndtools/bnd/blob/9e2ef8bcd4e541771f8dc3077ae3adca80fb6b29/biz.aQute.bndlib/src/aQute/bnd/build/defaults.bnd#L1-L39

Reply:
This is indeed the case.